### PR TITLE
Improves readme notes

### PR DESCRIPTION
**Pull request by MaxIsJoe**

### CODE REVIEW

**Comment by CorruptComputer** [comment]
Looks like the "Build with resentment" and "Contains technical debt" images are dead.

While you're updating this README could you remove these as well?
image

**Comment by corp-0** [comment]
why remove those?

**Comment by CorruptComputer** [comment]
Because the website hosting them is gone, the domain registration expired and someone could easily buy it and host whatever malicious content they want to.

https://www.whois.com/whois/forthebadge.com

**Comment by MaxIsJoe** [comment]
fixed them

File: README.md [hunk 1]
```diff
@@ -30,10 +31,10 @@ Read our Quick Start guide [here](https://unitystation.github.io/unitystation/co
 6. Submit a pull request
 
 > **Warning**
-> The Unitystation editor project greatly prefers a SSD, so it is recommended to have an SSD when developing
+> The Unitystation project **requires** an SSD to run in the unity editor.
```
Comment (Bod9001) [inline README.md]: like I said before it doesn't require an SSD , just for the best dev experience in SSD is recommended, 

require = it does not physically run if you don't have it
highly recommended is better for this scenario

Comment (corp-0) [reply README.md]: I think I agree

Comment (MaxIsJoe) [reply README.md]: if someone complains again about the editor being unusable while having an HDD I'm changing this back

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**